.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -43,19 +60,26 @@ run_sanity_check() {
     (
         cd release/util
         python sanity_check.py
+        bash sanity_check_cpp.sh
     )
     conda deactivate
 }
 
 _clean_up() {
+    bazel clean || true
     rm -rf "$TMP_DIR"
 }
 
 # Create tmp directory unique for the run
 TMP_DIR=$(mktemp -d "$HOME/tmp.XXXXXXXXXX")
```

Comment:
also, please double quote `$(mktemp ...)`, as into `"$(mktemp ...)"`

Reply:
fixed

Reply:
I don't think the new format fixes change any behavior but here is another run on BK: https://buildkite.com/ray-project/release-automation/builds/60